homeworks/magurmarian/hw01/calculation.py

```diff
@@ -0,0 +1,46 @@
+import argparse
+import sys
+
+from pyats.aetest import Testcase, test, main
+from somefunc import add, divide
+
+parameters = {
+    'num1': 3,
+    'num2': 0
+}
+
+
+class SmokeTest(Testcase):
+
+    @test
+    def test_add(self, steps, num1, num2):
+        with steps.start('Result bigger then 0') as step:
+            if add(num1, num2) > 0:
+                assert add(num1, num2) > 0
+            else:
+                self.skipped('Result is less then 0)')
+
+    @test
+    def test_devide(self, steps, num1, num2):
```

Comment:
Possibly a typo in function name.
```suggestion
    def test_divide(self, steps, num1, num2):
```